tests/executors/test_executor.py

```diff
@@ -1,18 +1,30 @@
 """Test basic executor functionality."""
+import gc
 import sys
 import time
 import signal
 import shlex
+import uuid
+from subprocess import check_output
 
 import pytest
 
 from mirakuru import Executor, HTTPExecutor
 from mirakuru.base import StartCheckExecutor
-from tests import test_server_path
+from tests import test_server_path, sample_daemon_path
 
 sleep_300 = 'sleep 300'
 
 
+def ps_aux():
```

Comment:
I've created 'signals' module because they were used in two other modules.
`ps_aux()` function is used only in here so I see no point in creating separate module right now.